app/views/users/index.html.erb

```diff
@@ -1,14 +1,4 @@
-<p style="color: green"><%= notice %></p>
-
-<h1>Users</h1>
-
-<div id="users">
-  <% @users.each do |user| %>
-    <%= render user %>
-    <p>
-      <%= link_to "Show this user", user %>
-    </p>
-  <% end %>
+<div data-controller="story_users">
+  <div class="field" data-story_users-target="content_u" >
+  </div>
```

Comment:
Same here, what is the purpose of this empty div?

Reply:
With the JS I create and add elements it depends on how many stories user has

Reply:
Okay, just write a descriptive comment in both div's for users and stories